dist/AjaxDynamicDataTable.js

```diff
@@ -0,0 +1,78 @@
+function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DynamicDataTable from "./DynamicDataTable";
+import { AxiosInstance as axios } from "axios";
+
+class AjaxDynamicDataTable extends Component {
+  constructor(props) {
```

Comment:
There is no need to pass props to the constructor unless using it to set initial state

Reply:
Duplicate.